<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabModel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Модель</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelModel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabParams">
       <attribute name="title">
        <string>Параметры</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayoutParams">
          <item row="5" column="2">
           <widget class="QLabel" name="labelK0">
            <property name="text">
             <string>K0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelX">
            <property name="text">
             <string>Иксы</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLineEdit" name="lineEditK0"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditStep"/>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QLabel" name="labelW">
            <property name="text">
             <string>Управление</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelB">
            <property name="text">
             <string>Ограничение (В)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelT">
            <property name="text">
             <string>Время (T)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelQ">
            <property name="text">
             <string>Количество слоев (q)</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="4">
           <widget class="QPushButton" name="pushButtonStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEditM2"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxNumLay"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineEditTime"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditM1"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelM2">
            <property name="text">
             <string>M2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelE">
            <property name="text">
             <string>Точность</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEditAccur"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelA">
            <property name="text">
             <string>Шаг градиентного спуска</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelM1">
            <property name="text">
             <string>M1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxNumNeur"/>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditLimit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelN">
            <property name="text">
             <string>Количество  нейронов (n)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QLineEdit" name="lineEditU0"/>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="labelU0">
            <property name="text">
             <string>U0</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2" colspan="2">
           <widget class="QTableWidget" name="tableWidgetControl">
            <column>
             <property name="text">
              <string>U</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="5" column="0" rowspan="3" colspan="2">
           <widget class="QTableWidget" name="tableWidgetInitValues">
            <column>
             <property name="text">
              <string>X0</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>A</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>W</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTables">
       <attribute name="title">
        <string>Таблицы</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidgetTables">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabFinalX">
           <attribute name="title">
            <string>Иксы</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTableWidget" name="tableWidgetFinalX"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabControl">
           <attribute name="title">
            <string>Управление</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QTabWidget" name="tabWidgetControl">
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabI">
           <attribute name="title">
            <string>Страница</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QTextBrowser" name="textBrowserLog"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGraphs">
       <attribute name="title">
        <string>Графики</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabX">
           <attribute name="title">
            <string>X</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCustomPlot" name="customPlotX" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabW">
           <attribute name="title">
            <string>W</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCustomPlot" name="customPlotW" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuParameters"/>
   <addaction name="menuRun"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
